Tests/DataPipeline/DataPipelineInteractionTests.swift

```diff
@@ -427,10 +423,10 @@ class DataPipelineInteractionTests: UnitTest {
     func test_registerDeviceToken_givenProfileIdentified_expectStoreAndRegisterDevice() {
         let givenDeviceToken = String.random
         let givenIdentifier = String.random
-        let givenDefaultAttributes: [String: Any] = ["foo": "bar"]
-        let expectedAttributes = givenDefaultAttributes.mergeWith([
-            "last_used": dateUtilStub.givenNow
```

Comment:
The test was broken after recent PR merge due to change in expectations from server side, removed `last_used` to fix test as server no longer requires it.

Reply:
Thanks for adding a comment here explaining this. 

> as server no longer requires it.

I understand the API may not require it, but I believe our SDK still sends this parameter. Therefore, I think that we should continue to test it if our SDK does send it. 

Although, looking at the test function here, the payload of `givenDefaultAttributes` looks small in size. Making me think that this test is not meant to test the accuracy of what the SDK payload sends, but is instead testing something else?